validate-form.js

```diff
@@ -0,0 +1,16 @@
+const form = document.querySelector('.cont-inp-form');
+const errorField = document.querySelector('.email-validat-mess');
+const emailField = document.querySelector('.user-email');
+
+form.addEventListener('submit', (event) => {
+  let errorMessage = null;
+  const lowerCaseEmail = emailField.value.toLowerCase();
+  if (lowerCaseEmail !== emailField.value) {
+    errorMessage = 'Email field should be in lowercase';
+  }
+
+  if (errorMessage !== null) {
+    event.preventDefault();
+    errorField.textContent = errorMessage;
+  }
+});
```

Comment:
- [ ] If a user submits the contact form with the email field containing uppercase characters an error message should show up. Although I couldn't submit the form with a capitalized email, I didn't see any error message. Please make the user see the appropriate error in this regard. Kindly let the error message meet all the neccessary requirements as specified in the instruction. :pray: